include/arch/arm/aarch64/timer.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2019 Carlo Caione <ccaione@baylibre.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_ARCH_ARM_AARCH64_TIMER_H_
+#define ZEPHYR_INCLUDE_ARCH_ARM_AARCH64_TIMER_H_
+
+#ifndef _ASMLANGUAGE
+
+#include <drivers/timer/arm_arch_timer.h>
+#include <zephyr/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define ARM_ARCH_TIMER_IRQ	((ARM_TIMER_VIRTUAL_IRQ + 1) << 8)
```

Comment:
we can get this from dt .  Or we can use the absolute interrupt id for sec/ns phys timer ppi.

Reply:
> we can get this from dt .

For the DT we are trying to follow what's done on Linux where all the PPIs for the S/NS/VIRT/HYP timers are specified and what is going to be actually used is left to the implementation / driver. So I don't think getting this from DT is an option here.

> Or we can use the absolute interrupt id for sec/ns phys timer ppi.

Not sure I got this. Are you suggesting to use the PHYS timer instead of the VIRT one?

Reply:
Probably not a bad idea switching to the phys timer anyway. Changing that in V3.

Reply:
Thanks.  I meant the same.